src/audio/audio.c

```diff
@@ -50,6 +51,73 @@ void *gTitleMusicChannel = 0;
 rct_vehicle_sound gVehicleSoundList[AUDIO_MAX_VEHICLE_SOUNDS];
 rct_vehicle_sound_params gVehicleSoundParamsList[AUDIO_MAX_VEHICLE_SOUNDS];
 rct_vehicle_sound_params *gVehicleSoundParamsListEnd;
+int gVolumeAdjustZoom = 0;
+
+int _volumeAdjust[SOUND_MAXID] = {
```

Comment:
It might be worth adding a comment column with all the sound ID constants like I tend to do with ride type constant arrays.